webapp/src/components/team_sidebar/index.js

```diff
@@ -1,6 +1,6 @@
 import {connect} from 'react-redux';
 
-import TeamSidebar from './team_sidebar.jsx';
+import TeamSidebar from './team_sidebar.tsx';
 
 function mapStateToProps(state) {
     const members = state.entities.teams.myMembers || {};
```

Comment:
If we converted this file to typescript, that wouldn't help much for this PR. There is a PR open for converting the redux store to typescript https://github.com/mattermost/mattermost-plugin-gitlab/pull/462, which would allow this `index.js` file to be properly typed as typescript.